accounts/models.py

```
@@ -1,27 +1,48 @@
-from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.models import AbstractUser, BaseUserManager
 from django.db import models
 import uuid
 
 from core.models import BaseModel
 
 
+class UserManager(BaseUserManager):
+    def create_user(self, username, email, password=None, **extra_fields):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_user`

(ANN201)